### PR TITLE
deploygate--upload-app-bitrise-step 1.1.2

### PR DESCRIPTION
**Pull request by jiikko**
TagCheck

### What to do if the build fails?

At the moment contributors do not have access to the CI workflow triggered by StepLib PRs. In case of a failed build, we ask for your patience. Maintainers of Bitrise Steplib will sort it out for you or inform you if any further action is needed.

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by jiikko** [comment]
This change is due to the previously announced "Remove Application Visibility".

en: https://medium.deploygate.com/public-profile-page-discontinued-33a28a97e606
ja: https://medium.deploygate.com/public-profile-page-discontinued-955288a43557

Currently, all apps are private on DeployGate. If you send the app-visibility parameter, it is ignored by the server.
Therefore, there are no side effects due to the removal of the app-visibility parameter.

File: steps/deploygate--upload-app-bitrise-step/1.1.2/step.yml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+    summary: (iOS only) A setting of true disables Push notification emails.
+    title: 'DeployGate: Disable Notify'
+    value_options:
+    - "true"
+    - "false"
+outputs:
+- DEPLOYGATE_UPLOAD_APP_STEP_RESULT_JSON: null
+  opts:
+    description: The raw json which Upload API has returned
+    summary: The raw json which Upload API has returned. Available iff the request
```
Comment (lpusok) [inline steps/deploygate--upload-app-bitrise-step/1.1.2/step.yml:95]:
```suggestion
    summary: The raw json which Upload API has returned. Available if the request
```
Please fix typo in a new release.

**Review by lpusok — APPROVED** [review]

**Comment by lpusok** [comment]
Hello @jiikko,
Thanks for the update. Can you please fix the typo above and create a new release?
Thanks.

**Comment by jiikko** [comment]
@lpusok 
Thank you for your review.

I got it. I will create a new release (deploygate--upload-app-bitrise-step **1.1.3** ) that just fixes the typo.

File: steps/deploygate--upload-app-bitrise-step/1.1.2/step.yml (hunk 1 above)
Comment (jiikko) [reply steps/deploygate--upload-app-bitrise-step/1.1.2/step.yml:95]: @lpusok 
No, it's not a typo. iff means if and only if.

**Pull request merged**